roles/openshift_metering/files/metering-helm-operator-deployment.yaml

```diff
@@ -0,0 +1,98 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: metering-helm-operator
+  labels:
+    app: metering-helm-operator
+spec:
+  replicas: 1
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      app: metering-helm-operator
+  template:
+    metadata:
+      labels:
+        app: metering-helm-operator
+    spec:
+      securityContext:
+        runAsNonRoot: true
+      containers:
+      - name: metering-helm-operator
+        image: "quay.io/coreos/chargeback-helm-operator:0.6.1-latest"
```

Comment:
should this value be configurable?

Reply:
It is, it's replaced using yedit if `openshift_metering_operator_image` is set.

Reply:
Is it not possible to provide this file as a jinja template instead? Then you could specify the value of the image as a `var` and it could default to the value above.

I know that with the service catalog there were times it wasn't possible to use a template since the files were actually sourced from another repo. Not sure if that is the same case here.

Reply:
It's possible, it's just more work than is needed since these files are just copied from another repo now, converting to Jinja means doing the conversion to Jinja anytime the upstream manifests change, so using yedit is a decent way to avoid that when we don't need to parameterize much.

Reply:
Makes sense